onnx/checker.cc

```diff
@@ -336,8 +347,7 @@ void check_model(const ModelProto& model) {
   for (const auto& opset_import : model.opset_import()) {
     opset_imports[opset_import.domain()] = static_cast<int>(opset_import.version());
   }
-  auto dit = opset_imports.find(ONNX_DOMAIN);
-  if (dit == opset_imports.end()) {
+  if (opset_imports.empty()) {
```

Comment:
This doesn't seem right to me. Do you want to avoid the check when you have other domain?

Reply:
The following checks are not necessary if at least one domain exists because (1) opset_import is good with IR>=3 (2) default opset_imports should be set only if opset_import is empty.

Reply:
I think the right check should be:
```
if (model.ir_version() >= 3) {
  // enforce opset_imports is not empty
} else {
  // enforce opset_imports is empty,
  // and set the default value for ONNX_DOMAIN
}
```

Reply:
Looks better.